packages/framework/src/Console/Commands/MakePublicationTagCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyde\Console\Commands;
+
+use Hyde\Console\Commands\Interfaces\CommandHandleInterface;
+use Hyde\Console\Concerns\ValidatingCommand;
+use Hyde\Framework\Features\Publications\PublicationService;
+use Hyde\Hyde;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use function Safe\json_decode;
+use function Safe\json_encode;
+use function Safe\file_put_contents;
+
+/**
+ * Hyde Command to create a new publication type.
+ *
+ * @see \Hyde\Framework\Testing\Feature\Commands\MakePublicationTypeCommandTest
+ */
+class MakePublicationTagCommand extends ValidatingCommand implements CommandHandleInterface
+{
+    /** @var string */
+    protected $signature = 'make:publicationTag';
+
+    /** @var string */
+    protected $description = 'Create a new publication type tag definition';
+
+    public function handle(): int
+    {
+        $this->title('Creating a new Publication Type Tag!');
+
+        $filename = Hyde::pathToRelative('tags.json');
+        $tags     = PublicationService::getAllTags();
+        $tagName  = $this->askWithValidation('name', 'Tag name', ['required', 'string']);
+        if (isset($tags[$tagName])) {
+            $this->output->error("Tag [$tagName] already exists");
+            return Command::FAILURE;
+        }
+
+        $lines = [];
+        $this->output->writeln('<bg=magenta;fg=white>Enter the tag values (end with an empty line):</>');
```

Comment:
![image](https://user-images.githubusercontent.com/95144705/204245223-f4d40c3f-a4da-4818-91d1-ae15565bced9.png)

It's not terminating for me, I think it might be because I'm on Windows and for some reason I truly don't understand Windows uses "\r\n" instead of "\n" for terminating newlines.